bin/cli.js

```diff
@@ -193,18 +209,21 @@ const getConfig = () => {
 
   config.files = merge(config.files, program.args);
   config.files = loadFiles(config.files);
-
   config.exclude = merge(config.exclude, program.exclude);
   config.files = exclude(config.files, config.exclude);
   config.nodeOnly = merge(config.nodeOnly, program.nodeOnly);
   config.browserOnly = merge(config.browserOnly, program.browserOnly);
+  config.verbosity = program.verbose > 3 ? 3 : program.verbose;
+  config.quiet = program.quiet;
 
   if (config.environments.includes('browser')) {
     config.browser = config.browser || {};
-
-    config.browser.browsers = merge(
-      config.browser.browsers, program.browsers);
-    config.browser.logLevel = program.browserLog || config.browser.logLevel;
+    config.browser.browsers = merge(config.browser.browsers, program.browsers);
+    config.browser.logLevel =
+      config.quiet ? 'disable' :
+        program.browserLog ||
+        Object.keys(logLevels)[program.verbose] ||
```

Comment:
Same remark as in https://github.com/metarhia/metatests/pull/94#discussion_r225298769.